charts/azure-apps/examples/secrets-store-csi-driver-provider-azure.yaml

```diff
@@ -0,0 +1,8 @@
+secretsStoreCsiDriverAzure:
+  enabled: true
+  project: infra-secrets-store-csi-azure
+  values:
+    # Install RBAC roles and bindings required for K8S Secrets syncing if true
+    syncSecret:
```

Comment:
This stuff should be under `secrets-store-csi-driver` as shown [here](https://github.com/Azure/secrets-store-csi-driver-provider-azure/blob/master/charts/csi-secrets-store-provider-azure/values.yaml#L98)

Reply:
Missed it, thanks!